### PR TITLE
add contributing description update readme

### PR DESCRIPTION
**Pull request by dominikhajduk**
@php-refactor please look on proposed changes and comment (we can merge after 2 confirmation that changes are ok)

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+
+If you'd like to contribute, please read the following:
+
+* Contributing for mail sender is typical for [Open Source projects][1]
+
+* You need [fork project][2]
+
+* [Run project locally][3] 
+
+* Do changes on your forked repository and pull them
```
Comment (partikus) [inline CONTRIBUTING.md]: ``push them`` instead ``pull``

**Comment by dominikhajduk** [comment]
@partikus fixed, thanks

**Comment by PorkOne** [comment]
:thumbsup:

**Comment by partikus** [comment]
:+1:

**Pull request merged**